core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -155,11 +154,11 @@
         </li>
         <li>
           <ul class="nav oppia-navbar-tabs-admin">
-            <li uib-dropdown class="nav-item uib-dropdown float-right">
+            <li uib-dropdown class="nav-item uib-dropdown float-right" ng-class="{'open' : $ctrl.profileDropdownIsActive}">
               <a uib-dropdown-toggle class="nav-link uib-dropdown-toggle oppia-navbar-dropdown-toggle"
                  data-toggle="dropdown"
-                 ng-mouseover="$ctrl.onMouseoverProfilePictureOrDropdown($event)"
-                 ng-mouseleave="$ctrl.onMouseoutProfilePictureOrDropdown($event)">
+                 ng-mouseenter="$ctrl.profileDropdownIsActive = true"
```

Comment:
Any specific reason for replacing `mouseover` with  `mouseenter`?

Reply:
Nothing specific,
In general, we pair mouse-over and mouse-out same as mouse-enter with mouse-leave,
I'll remove it if not needed.